docs/src/description/organization.md

```diff
@@ -0,0 +1,77 @@
+# Code organization
+
+AirScript project can consist of one or more modules, each module located in a separate file with a `.air` extension. For projects consisting of multiple modules, one module must be declared as the root module, and all other modules must be declared as supporting modules.
+
+Currently, all modules must be located in a single directory, but in the future this limitation will be removed.
+
+All modules must start with a module name declaration followed by a set of source sections. These sections describe both the metadata and constraint evaluation logic for the AIR. Depending on the module type, some source sections may be required, be optional, a may not be allowed. The table below summarizes this information.
```

Comment:
The part about source sections only applies to root modules really, as library modules only require the module declaration to be valid, and can contain any number of constant, function, or evaluator declarations, as well as imports. Might be worth distinguishing that here.